R/curl.R

```diff
@@ -23,7 +29,8 @@
 #' curl_translate("curl http://example.com -X DELETE")
 #' curl_translate("curl http://example.com --header A:1 --header B:2")
 #' curl_translate("curl http://example.com --verbose")
-curl_translate <- function(cmd) {
+curl_translate <- function(cmd,
+                           simplify_headers = TRUE) {
```

Comment:
```suggestion
curl_translate <- function(cmd, simplify_headers = TRUE) {
```

Since it fits on one line.